### PR TITLE
Added and styled navbar

### PR DESCRIPTION
**Pull request by ewelina-slepko**

### CODE REVIEW

File: src/app/home/panel/navbar/navbar.component.html [hunk 1]
```diff
@@ -1 +1,23 @@
-<p>navbar works!</p>
+<div class="navbar">
+  <div class="element-container">
+    <i class="fas fa-home navbar-icon"></i>
```
Comment (FilipTalaga) [inline src/app/home/panel/navbar/navbar.component.html:3]: Did you know [this](https://www.w3schools.com/tags/tag_i.asp)? :D

File: src/app/home/panel/navbar/navbar.component.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  height: 60px;
+  display: flex;
+  flex-direction: column;
+}
+
+.navbar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 99.99%;
```
Comment (FilipTalaga) [inline src/app/home/panel/navbar/navbar.component.scss:14]: What? Why?

File: src/app/home/panel/navbar/navbar.component.scss [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  height: 60px;
+  position: fixed;
+  max-width: 26rem;
+  bottom: 0;
+  border: 1.4px solid $medium-gray;
```
Comment (FilipTalaga) [inline src/app/home/panel/navbar/navbar.component.scss:19]: Do not use fractions on pixels. Screens can only display pixels or not, there is no such thing as 0.4 of a pixel.

File: src/app/home/panel/navbar/navbar.component.scss [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  background-color: $white;
+  -webkit-box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.18);
+  -moz-box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.18);
+  box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.18);
```
Comment (FilipTalaga) [inline src/app/home/panel/navbar/navbar.component.scss:23]: Box-shadow should be enough. The rest of them would actually be added by SASS.

File: src/app/home/panel/navbar/navbar.component.scss (hunk 3 above)
Comment (ewelina-slepko) [reply src/app/home/panel/navbar/navbar.component.scss:19]: ok

**Pull request merged**